FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Create.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using AutoFixture;
+using FrontDesk.Core.Aggregates;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Create
+  {
+    private Fixture _fixture = new Fixture();
+
+    [Fact]
+    public void CreateSuccess()
+    {
+      var scheduleId = Guid.NewGuid();
+      var appointment = Appointment.Create(scheduleId, 1, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test");
+
+      Assert.Null(appointment.DateTimeConfirmed);
+      Assert.Equal(1, appointment.AppointmentTypeId);
+      Assert.Equal(scheduleId, appointment.ScheduleId);
+      Assert.Equal(1, appointment.ClientId);
+      Assert.Equal(1, appointment.DoctorId);
+      Assert.Equal(1, appointment.PatientId);
+      Assert.Equal(1, appointment.RoomId);
+      Assert.Equal(3 * 60, appointment.TimeRange.DurationInMinutes());
+      Assert.Equal("Title Test", appointment.Title);
+    }
+
+    [Fact]
+    public void ThrowExceptionZeroClientId()
+    {
+      var scheduleId = Guid.NewGuid();
+
+      Assert.Throws<ArgumentException>(() =>
+        Appointment.Create(scheduleId, 0, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test"));
+    }
+
+    [Fact]
+    public void ThrowExceptionNegativeClientId()
+    {
+      var scheduleId = Guid.NewGuid();
+
+      Assert.Throws<ArgumentException>(() =>
+        Appointment.Create(scheduleId, -1, 1, 1, DateTime.Now, DateTime.Now.AddHours(3), 1, 1, "Title Test"));
```

Comment:
The Create method will be a lot easier to read if you're passing in named variables. You can use class level constants for the default valid values. Then in each test of an invalid value you can use a local like `int invalidClientId = -1;` and then pass that. Makes the tests easier to read and less error-prone.